python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -398,11 +398,17 @@ def main():
         exit_code = 16
 
     try:
-        import scandir
-        scandir.__file__
+        import _scandir
     except:
-        sys.stderr.write('"scandir" missing.\n')
+        sys.stderr.write('"_scandir" missing.\n')
         exit_code = 17
+    else:
+        if not os.getcwd() in _scandir.__file__:
```

Comment:
it would help to extract this into a helper function which we can use to test any other other module... as we want similar tests for all the other .so/C modules